tensorflow/lite/kernels/pooling_test.cc

```diff
@@ -147,6 +231,126 @@ TEST(QuantizedPoolingOpTest, AveragePool) {
   EXPECT_THAT(m.GetOutput(), ElementsAreArray({44, 92}));
 }
 
+TEST(QuantizedPoolingOpTest, AveragePoolActivationRelu) {
+  // Choose the input ranges carefully so that the dequantized output matches
```

Comment:
Instead of just mentioning choosing range carefully, could you please also comment on the scale and zero point when using (0, 15.9375) for both uint8 and int8?

Also, when (0, 15.9375) is used, what is the diff in line 249?

Thanks.

Reply:
@jianlijianli : The diff is because of improper input [min, max], now i have modified the test scripts as per your comments, please review, Thanks!